mungegithub/github/github.go

```diff
@@ -1562,12 +1562,11 @@ func (obj *MungeObject) GetPR() (*github.PullRequest, bool) {
 		return obj.pr, true
 	}
 	if !obj.IsPR() {
-		glog.Errorf("Issue: %d is not a PR", *obj.Issue.Number)
 		return nil, false
```

Comment:
Shouldn't GetPR be changed so that 'isn't a PR' can be distinguished from 'github API error' as you noted in a previous comment? (i.e. change this statement to 'return nil, true')